packages/mockserver-config-writer/src/mockserver-config/ui5-mock-yaml.ts

```diff
@@ -5,7 +5,7 @@ import type { CustomMiddleware } from '@sap-ux/ui5-config';
 import type { Manifest } from '@sap-ux/project-access';
 import type { Ui5MockYamlConfig } from '../types';
 import type { MockserverConfig } from '@sap-ux/ui5-config/dist/types';
-import { getMainServiceDataSource } from '../app-info';
+import { ODataSourceType, getMainServiceDataSource, getODataSources } from '../app-info';
```

Comment:
Instead of `ODataSourceType` we should also use existing type `ManifestNamespace.DataSourceEnum['type']`